plugins/item_licenses/web_client/templates/item_licenses_config.jade

```diff
@@ -0,0 +1,11 @@
+.g-config-breadcrumb-container
+p.g-item-licenses-description
+  | Configure the available licenses for items.
+form#g-item-licenses-settings-form(role="form")
+  .form-group
+    label.control-label(for="g-item-licenses") Licenses
+    textarea#g-item-licenses.form-control= licenses
```

Comment:
If this plugin becomes popular, we'll want to make this a more user-friendly series of widgets, but this is fine for now.